assembly/std/utils/arrays.ts

```diff
@@ -0,0 +1,11 @@
+export function staticArrayToUint8Array(
+  staticArray: StaticArray<u8>,
+): Uint8Array {
+  let uint8Array = new Uint8Array(staticArray.length);
+
+  for (let i = 0; i < staticArray.length; i++) {
+    uint8Array[i] = staticArray[i];
+  }
+
+  return uint8Array;
```

Comment:
What about using something like:
```typescript
const arr = new Uint8Array(staticArray.length);
memory.copy(changetype<usize>(arr), changetype<usize>(staticArray), arr.length);
return arr;
```

Reply:
I've moved the function in the project because it could be useful elsewhere.